tools/ci/github-runners/v3/Dockerfile

```diff
@@ -1,40 +0,0 @@
-FROM rust:latest
-
-RUN apt-get update && apt-get install --no-install-recommends --yes \
-  clang \
-  cmake \
-  git-lfs \
-  libasound2-dev \
-  libdbus-1-dev \
-  libhdf5-dev \
-  libluajit-5.1-dev \
-  libopusfile-dev \
-  libsystemd-dev \
-  libudev-dev \
-  lua-inspect \
-  mkdocs \
-  mkdocs-material \
-  ninja-build \
-  python3 \
-  python3-click \
-  python3-git \
-  rsync \
-  zstd \
-  && rm --recursive --force /var/lib/apt/lists/*
-
-RUN rustup component add clippy rustfmt
-
-RUN useradd --no-log-init --create-home --home-dir /ci ci
-WORKDIR /ci
-USER ci:ci
-
-ARG SDK_VERSION
-RUN wget --no-verbose https://github.com/HULKs/meta-nao/releases/download/${SDK_VERSION}/HULKs-OS-toolchain-${SDK_VERSION}.sh && \
-  chmod +x HULKs-OS-toolchain-${SDK_VERSION}.sh && \
-  ./HULKs-OS-toolchain-${SDK_VERSION}.sh -y -d /ci/.naosdk/${SDK_VERSION} && \
-  rm HULKs-OS-toolchain-${SDK_VERSION}.sh
-
-COPY cargo.sh.patch /ci
-RUN cd /ci/.naosdk/${SDK_VERSION}/sysroots/x86_64-naosdk-linux/environment-setup.d && \
-  patch -i /ci/cargo.sh.patch && \
-  rm /ci/cargo.sh.patch
```

Comment:
Is the `cargo.sh.patch` still being used? It looks like it is dead code currently. :thinking:

Reply:
no it is not. Removed it...

Reply:
Did you check whether it was still used or did you check what it did and decided that it is obsolete? Vote for an intentional instead of coincidental decision. If the latter, feel free to resolve.

Reply:
This is intentional: This patch previously moved the cargo home to something outside the container to persist and cache the registry. As the entire SDK now lives outside the container, also the default cargo home at `$OECORE_TARGET_SYSROOT/home/cargo` is outside the container.

Reply:
I even think,  it is more correct now, as the cargo homes are not shared between different SDK versions.